app/assets/stylesheets/main.scss

```diff
@@ -22,14 +22,46 @@ $custom-border: 5px dashed black;
 
 
 body{
-h1{
+  h1{
 
-  border: $custom-border;
+    border: $custom-border;
     text: {
       align: center;
       color: $color3;
     }
+  }
+}
+
+.form-outline {
```

Comment:
based on the HTML and CSS best practice lesson, should we move this to a partial so we can use the same styling for each form? also can we create a @mixin in our Sass so we can access in more places

Reply:
Emily that's a great thought. I lean towards implementing this as is and then adding it to our list of things to refactor!